scripts/_helpers.py

```diff
@@ -1241,104 +1238,82 @@ def get_GADM_layer(country_list, layer_id, update=False, outlogging=False):
         When the requested layer_id is greater than the last available layer, then the last layer is selected.
         When a negative value is requested, then, the last layer is requested
     """
```

Comment:
Just to make sure: do we still maintain this functionality to have a safe fallback if an inappropriate layer id is requested? Not sure the check really needed here but probably worth to revise the docsting.

Reply:
This function is an utility to select the appropriate shapefile and correct the id when necessary.
Honestly, I believe we should drop it when we normalize the GADMs